### ISSUE TITLE
Explain manual deployment to Heroku

### ISSUE THREAD

**Issue by filipeximenes** [vintasoftware/django-react-boilerplate#161]
README does not talk about adding build packs before pushing the code to a new heroku instance

**Comment by hugobessa** [comment]
doesn't app.json cover this? is manual deployment explanation really necessary?

**Comment by aericson** [comment]
Agree with @hugobessa. We will have duplicated information and will be easy to update one and miss the other.

People doing a manual deploy should base it on `app.json` as stated in README.md.
```
P.S. if you want to deploy in a different way please check the app.json file for what needs to be configured.
```

**Comment by filipeximenes** [comment]
Alright, so we should at least explain how this heroku button works. The docs are not easy to understand.

**Comment by embs** [comment]
I've just deployed a new boilerplate-based app to Heroku [via cURL with app.json](https://devcenter.heroku.com/articles/setting-up-apps-using-the-heroku-platform-api) and had to:

- add ALLOWED_HOSTS
- add credit card to my Heroku account
- add `requests` dependency to Pipfile

**Comment by embs** [comment]
Maybe we could change this issue title from

> Explain manual deployment to Heroku

to

> Explain deployment to Heroku

**Comment by joaopslins** [comment]
This is the thing about the heroku button, it expects a github repository as a template. Then it tries to find an app.json in that repo.

Heroku button URL looks like this: `https://www.heroku.com/deploy/?template=https://github.com/heroku/node-js-getting-started`

Since the boilerplate project does not cover setting up a repo on github, we cannot craft this URL beforehand... Unless we use the boilerplate repo itself in it.

**Comment by joaopslins** [comment]
> I've just deployed a new boilerplate-based app to Heroku [via cURL with app.json](https://devcenter.heroku.com/articles/setting-up-apps-using-the-heroku-platform-api) and had to:
> 
> * add ALLOWED_HOSTS
> * add credit card to my Heroku account
> * add `requests` dependency to Pipfile

@embs I just have created a test project on heroku using the boilerplate. Assuming that you already have a heroku account set, I just needed to add ALLOWED_HOSTS, because that's a value that is not autogenerated. It depends on the heroku project name. Maybe we should add this on the instructions as well.

**Comment by joaopslins** [comment]
Maybe implicit template is the way:

https://devcenter.heroku.com/articles/heroku-button#using-an-implicit-template

**Comment by joaopslins** [comment]
Fixed by #372
